### PR TITLE
#84 - Added ResourceFromSlice

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #84 
Added `ResourceFromSlice` class required for interoperation between `AuthSlice` class from HTTP module and `Resource` classes in NuGet repository

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#87](https://github.com/artipie/nuget-adapter/issues/87) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/8a7c6a1d-803d-4835-8f72-aa8187a9b001); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/nuget/http/ResourceFromSliceTest.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for {@link ResourceFromSlice}.
+ *
+ * @since 0.2
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+class ResourceFromSliceTest {
```
[the remark this answers is not in the document]
Comment (olegmoz) [reply src/test/java/com/artipie/nuget/http/ResourceFromSliceTest.java:49]: @victornoel thank you, fixed

**Comment by victornoel** [comment]
@g4s8 I'm not clear what is the context of this PR but in terms of code it looks good to me

**Comment by 0crat** [comment]
Job was finished in 19 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@victornoel please make sure you will find at least 3 issues during next CR, thanks!

**Comment by victornoel** [comment]
@sereshqua yep, sorry about that

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
> @0crat quality acceptable ([here](https://github.com/artipie/nuget-adapter/issues/87#issuecomment-633933768))

@sereshqua There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@2be63c13-446f-4aaf-b035-a2d81942b365, thread: PQ-CT2E6TK9B
com.zerocracy.farm.reactive.StkRuntime[76] java.lang.IllegalStateException: org.codehaus.groovy.runtime.InvokerInvocationException in com.zerocracy.stk.pm.cost.pay_cash
org.codehaus.groovy.reflection.CachedMethod[105] org.codehaus.groovy.runtime.InvokerInvocationException: java.io.IOException: Failed to add transaction
com.zerocracy.pm.cost.Ledger[133] java.io.IOException: Failed to add transaction
org.postgresql.jdbc.PgConnection[767] org.postgresql.util.PSQLException: This connection has been closed.
```

[0.53.14](https://github.com/zerocracy/farm): CID: [ca0f9eac-b4ac-438d-ab0b-4bbc7128aeda](https://www.0crat.com/footprint/CT2E6TK9B/ca0f9eac-b4ac-438d-ab0b-4bbc7128aeda), Type: "Pay cash"

**Comment by 0pdd** [comment]
@olegmoz the puzzle [#203](https://github.com/artipie/nuget-adapter/issues/203) is still not solved.

**Comment by 0pdd** [comment]
@olegmoz the puzzle [#unknown]() is still not solved; solved: [#203](https://github.com/artipie/nuget-adapter/issues/203).

**Pull request merged**